docusaurus.config.js

```diff
@@ -120,17 +120,18 @@ module.exports = {
             {
               to: "cli/",
               label: "CLI",
-              // position: "left",
             },
             {
               to: "protocol/",
               label: "Protocol",
-              // position: "left",
+            },
+            {
+              to: "holder/",
+              label: "Holder",
```

Comment:
I think I would have expected the `Holder` section to be at the same level as `Developer` and `Validator`, wdyt?
Especially judging by the content on https://docs.celo.org/holder, which reads like a big entry page.

Reply:
`Developer` and `Holder` are at the same level we just tucked `Holder` into the more part in order to have a better mobile responsible navbar.

If we want `Holder` to be outside the dropdown then we have to trade it with `Validator` or `What is Celo` item.

<img width="706" alt="image" src="https://github.com/celo-org/docs/assets/38040789/e35074e5-bc33-4bb9-a419-bb923ddde98a">

<img width="1035" alt="image" src="https://github.com/celo-org/docs/assets/38040789/149e752b-3b98-4d0f-9549-865a8bfe9b30">

Reply:
That makes sense. I wouldn't trade it for one of the other options then.
How about making it the first in the dropdown?

Reply:
Done!